app/main/api/handlers/getWallet.ts

```diff
@@ -0,0 +1,39 @@
+import { asRuntimeType } from "app/common/runtimeTypes"
+import { SubSystems } from "app/main/system"
+import { WalletInfo, Wallet } from "app/common/runtimeTypes/storage/wallets"
+import createAesLevelStorage from "app/main/storage/jsonAesLevel"
+
+const searchWallet = (wallets: Array<WalletInfo>, id: string) =>
+    wallets.find((wallet: WalletInfo) => wallet.id === id)
+
+/**
+ * Handler function for "getWallet" method.
+ *
+ * @export
+ * @param {SubSystems} system
+ * @param {id, *} params
+ * @returns {Wallet} object taken from storage with provided id
+ */
+export default async function getWallet(system: SubSystems, params: any) {
```

Comment:
I just realized this handler function is hardly testable as it's hard coded to use `level` as persister (which has profound side effects, file system wise).

Maybe we'd better move the storage creation function (`createAesLevelStorage` in this case) into `system` or `system.walletStorage` so that we can use `jsonAesInMemory` or whatnot when testing.

cc/ @kronolynx